google/cloud/documentai_toolbox/wrappers/hocr.py

```diff
@@ -0,0 +1,221 @@
+import dataclasses
+from typing import List, Union
+from google.cloud import documentai
+
+from google.cloud.documentai_toolbox.constants import ElementWithLayout
+
+
+def _get_bounding_box(element_with_layout: ElementWithLayout, dimensions: documentai.Document.Page.Dimension):
+    if element_with_layout.layout.bounding_poly.vertices:
+        min_x, min_y = element_with_layout.layout.bounding_poly.vertices[0].x, element_with_layout.layout.bounding_poly.vertices[0].y
+        max_x, max_y = element_with_layout.layout.bounding_poly.vertices[2].x, element_with_layout.layout.bounding_poly.vertices[2].y
+        return f"bbox {int(min_x)} {int(min_y)} {int(max_x)} {int(max_y)}"
+    min_x, min_y = element_with_layout.layout.bounding_poly.normalized_vertices[0].x, element_with_layout.layout.bounding_poly.normalized_vertices[0].y
+    max_x, max_y = element_with_layout.layout.bounding_poly.normalized_vertices[2].x, element_with_layout.layout.bounding_poly.normalized_vertices[2].y
+    return f"bbox {int(min_x * dimensions.width)} {int(min_y * dimensions.height)} {int(max_x * dimensions.width)} {int(max_y * dimensions.height)}"
+
+
+def _get_text(element_with_layout: ElementWithLayout, document_text):
+    start_index = element_with_layout.layout.text_anchor.text_segments[0].start_index
+    end_index = element_with_layout.layout.text_anchor.text_segments[0].end_index
+    return document_text[start_index:end_index].replace('/n', '')
+
+
+def _load_hocr_words(words: List[documentai.Document.Page.Token], document_text: str, dimensions: documentai.Document.Page.Dimension):
```

Comment:
(1) It may be simpler to test if the method converts a single Document AI Token to a single hOCR Word, and let the caller handle the `for` loop.

(2) The function name could provide even more information, such as `hocr_words_from_documentai_tokens`, or the singular form (if you move the `for` loop out), `hocr_word_from_documentai_token`.

Side note: there are other alternatives that is more object-oriented, now that we have the classes.  For example

```
class _HocrWord:
  ...
  @classmethod
  def from_word(cls, word: wrappers.Word) -> '_HocrWord':
    ...
```
This still requires figuring out whether a "word" needs to know about the page's dimensions (which does not make sense to me), but indeed, the `text` should be already contained in the `word` and having to pass both in seems like a sign of something else not working properly.